android/src/main/java/com/pdftron/reactnative/views/DocumentView.java

```diff
@@ -4947,6 +4947,19 @@ public void downloadFailed(@NonNull Exception e) {
         });
     }
 
+    public void setFormFieldHighlightColor(ReadableMap fieldHighlightColor) throws PDFNetException {
+        if (getPdfViewCtrl() != null && fieldHighlightColor != null) {
+            double red = (double) (fieldHighlightColor.getInt(COLOR_RED) / 255f);
+            double green = (double) (fieldHighlightColor.getInt(COLOR_GREEN) / 255f);
+            double blue = (double) (fieldHighlightColor.getInt(COLOR_BLUE) / 255f);
+            double alpha = (double) (fieldHighlightColor.getInt(COLOR_ALPHA) / 255f);
+            ColorPt colorPt = new ColorPt(red, green, blue, alpha);
```

Comment:
ideally you have a `convertRGBAToColorPt` method where you can reuse it in various places

Reply:
new method convertRGBAToColorPt